### DISCUSSION TITLE
游戏文件图标

### DISCUSSION THREAD

**Discussion by rjdxxxxxccc** [Meloong-Git/PCL#3947] [提问]
游戏文件图标设置后在将这个文件制作成整合包后游戏图标显示启动器默认图标

**Comment by WorldHim** [comment]
抱歉没有听懂。能不能给出更具体的信息（比如截图

**Reply by rjdxxxxxccc** [reply]
游戏版本文件设置图标后以制作的整合包后，图标不是我设置的

**Comment by WForst-Breeze** [comment]
是意为在制作整合包时，在启动器内设置了版本图标后没有生效吗？

如果是的话，请尝试刷新版本列表，如若为题依旧存在，请附带截图和日志信息提交反馈，谢谢！

**Reply by rjdxxxxxccc** [reply]
卸载后重装图标恢复默认的
